diagnostic.md

```diff
@@ -5,74 +5,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The back end, or server side, is saved code that remains on the server that can be retrieved later.
 ```
```

Comment:
Yes, but just to clarify; it handles user requests and interacts with a database to store and retrieve data based on those user requests.